.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: "Release"
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency: release
+
+jobs:
+  release-charts:
+    name: Release Charts
+    runs-on: ubuntu-latest
+    
+    steps:
+      # Checkout everything since ct uses git diff to determine charts
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b #v3.0.2
```

Comment:
Non blocking. 

We should rethink this approach of manually pinning actions. 
We are loosing all the goodies that come with dependabot.

Is there a reason for pinning those along with docker images?

Reply:
just a good practice actually to not get something which can be vulnerable and still is not detected.

Reply:
It is recommended by Github Security Best Practices.
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Without auditing the source source code of the action (as described in the above documentation), I think this does not provide any actual benefit. 
But I am not strongly opinionated on this. 

It should be nice having dependabot running on our pipelines. It is actually thought of as of the best practises by security experts.

Reply:
https://github.blog/2022-04-28-best-practices-to-keep-your-projects-secure-on-github/#respond-to-vulnerable-dependencies